packages/node_modules/@webex/internal-plugin-support/src/support.js

```diff
@@ -63,8 +63,7 @@ const Support = WebexPlugin.extend({
 
         return this.webex.upload({
           file: logs,
-          api: 'atlas',
-          resource: 'logs/url',
+          uri: 'https://client-logs-intb.ciscospark.com/api/v1/logs',
```

Comment:
Is there a service we can use for this instead of hard coding intb? Or maybe an ENV var is needed?

Reply:
there is a clientLogs service but I didn't see it in the wdm registration payload